release/models/aft/openconfig-aft.yang

```diff
@@ -232,6 +241,27 @@ module openconfig-aft {
         uses aft-ethernet-structural;
       }
 
+      container state-synced {
+        description
+          "AFT streaming over gNMI is an eventually consistent system.
```

Comment:
```suggestion
          "In some cases AFT streaming (e.g., over gNMI) is an eventually consistent system.
```

Minor editorial change because gNMI is not intended to be the only supported transport for OpenConfig models (albeit at this point it is the defacto standard around telemetry).